packages/api/src/lib/lms.ts

```diff
@@ -243,6 +243,16 @@ export enum BloomDimension {
   Evaluate = 'Evaluate',
   Create = 'Create',
 }
+export enum InsightType1 {
```

Comment:
These are not an LMS concept. This comes from querying MMT. So they should be in mmt.ts